galette/lib/Galette/IO/PdfMembersCards.php

```diff
@@ -308,13 +309,17 @@ public function drawCards(array $members): void
             $this->writeHTML('<strong>' . $email . '</strong>', false, false);
 
             // Lower colored strip with long text
+            $nb_car =  round($this->wi / iconv_strlen($this->preferences->pref_card_strip) * 4.1 );
+			if ( $nb_car > 12 ) {
+				$nb_car = 12 ;
+				}
             $this->SetFillColor($fcol['R'], $fcol['G'], $fcol['B']);
             $this->SetTextColor(
                 $this->tcol['R'],
                 $this->tcol['G'],
                 $this->tcol['B']
             );
-            $this->SetFont(self::FONT, 'B', 8);
+            $this->SetFont(self::FONT, 'B', $nb_car);
```

Comment:
Well, I'm not sure at all since the value is either arbitrary; or calculated a way I do not understand.